search_service/models/badge.py

```diff
@@ -0,0 +1,24 @@
+import attr
+from marshmallow_annotations.ext.attrs import AttrsSchema
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class Badge:
```

Comment:
should we create a model for tag which could be used for both tag and badge? and the `__repr__` could be different depends on the type?

Reply:
sure, happy to do that

Reply:
> sure, happy to do that

Would this one in `amundsen-common` support both Tag and Badge for Badge is just a Tag with type `badge`? We can reuse this one.
https://github.com/lyft/amundsencommon/blob/b52eae378eaec39cc538e969391513c4ccad59a6/amundsen_common/models/table.py#L22

Reply:
I used the model in amundsen-common.